leader-election-impl/src/main/java/com/palantir/leader/PaxosLeaderElectionService.java

```diff
@@ -345,7 +345,7 @@ private void proposeLeadership(LeadershipToken token) {
             proposer.propose(seq, null);
         } catch (PaxosRoundFailureException e) {
             // We have failed trying to become the leader.
-            leaderLog.warn("Leadership was not gained", e);
+            leaderLog.warn("Leadership was not gained");
```

Comment:
Happy to include it. 

However, I'm not convinced that information adds much signal.  As I understand the implementation, a `PaxosRoundFailureException` can be thrown if: 

 - in Phase 1, if too many nodes either: 
    - had their logs truncated, or
    - timed out
 - in Phase 2, if too many nodes either: 
    - responded to another prepare request with a higher pid, or
    - timed out 

In all of these cases, the sensible thing is to retry the leadership election. And regardless of which phase fails, it's not clear whether the failure was due to timeouts or some other reason. 

We plan to revisit use of this exception as part of a broader refactor in #444, and should get more explicit errors then.